NEWS.md

```diff
@@ -5,7 +5,7 @@ This major update and addresses a variety of comments made by reviewers from the
 The update introduces breaking changes. If you want to keep using the older version, you can download it using `remotes::install_github("epiforecasts/scoringutils@v1.2")`. 
 
 ## Package updates
-- In `score()`, required columns "true_value" and "prediction" were renamed and replaced by required columns "observed" and "predicted". Scoring functions now also use the function arguments "observed" and "predicted" everywhere consistently. 
+- In `score()`, required columns "true_value" and "prediction" were renamed and replaced by required columns "observed" and "predicted" and "model" (which now is a required column). Scoring functions now also use the function arguments "observed" and "predicted" everywhere consistently. 
```

Comment:
Note that before we hit go on the new version we might need to reorg this so that the flow is from as_forecast through the workflow.

Reply:
Yes completely agree - made an issue here: https://github.com/epiforecasts/scoringutils/issues/687